vars/main.yml

```diff
@@ -1,2 +0,0 @@
----
-platform_client_base_path: /var/www/platform-client
```

Comment:
I think this shouldn't be here since it shadows any value passed to the role by parameter